src/main/java/net/snowflake/ingest/connection/ServiceResponseHandler.java

```diff
@@ -24,7 +24,9 @@
 public final class ServiceResponseHandler {
   // Create a logger for this class
   private static final Logger LOGGER = LoggerFactory.getLogger(ServiceResponseHandler.class);
-
+  private enum ApiName{
+    InsertFiles, GetHistory, GetHistoryRange, ConfigureClient, GetClientStatus
```

Comment:
capitalised names since they are constants.

Reply:
One liners for each enum types. 
Lets remove verb from APIs. 
INSERT_FILES

INSERT_REPORT

LOAD_HISTORY_SCAN

CLIENT_CONFIGURE

CLIENT_STATUS

https://docs.snowflake.com/en/user-guide/data-load-snowpipe-rest-apis.html
Lets also have a private constructor. 
We can add various fields in it but it is not needed at the moment. (One of that I can think of is VERB -> GET, POST etc)